core/services/workflows/cmd/cre/utils/runner.go

```diff
@@ -100,11 +107,12 @@ var defaultCleanup = func(ctx context.Context, cfg RunnerConfig, registry *capab
 
 func DefaultHooks() *RunnerHooks {
 	return &RunnerHooks{
-		Initialize: defaultInitialize,
-		BeforeRun:  emptyHook,
-		AfterRun:   emptyHook,
-		Cleanup:    defaultCleanup,
-		Finally:    emptyHook,
+		Initialize:  defaultInitialize,
```

Comment:
Adding some doctoring
```suggestion
                // Initialize hook sets up resources used by the Runner
		Initialize:  defaultInitialize,
```